packages/ui/src/components/Topic/components/types.ts

```diff
@@ -57,4 +57,6 @@ export type CustomRetentionUnit =
 
 export type RadioSelectType = "day" | "week" | "custom" | "unlimited";
 
+export type CleanupPolicyType = "delete" | "compact" | "delete,compact";
```

Comment:
[We already have this](https://github.com/redhat-developer/rhosak-ui/blob/48ba69488a2a976c4e7b7fc9464aa9dece3c7524/packages/ui-models/src/models/topic-config.ts#L55), let's not duplicate things.